pkg/grafana/provider.go

```diff
@@ -79,5 +79,6 @@ func (p *Provider) GetHandlers() []grizzly.Handler {
 		NewDashboardHandler(p),
 		NewRuleHandler(p),
 		NewSyntheticMonitoringHandler(p),
+		NewContactPointHandler(p),
```

Comment:
Can we call it 
```suggestion
		NewAlertContactPointHandler(p),
```

So that it's grouped with https://github.com/grafana/grizzly/pull/286. Same for all other places where it's only `ContactPoint`. Maybe `AlertingContactPoint` and `AlertingRuleGroup` would be even better 🤔

Reply:
Renamed